3.5-windowsservercore-ltsc2016/runtime/Dockerfile

```diff
@@ -5,7 +5,8 @@ FROM microsoft/dotnet-framework:3.5-windowsservercore-ltsc2016
 RUN powershell -Command Add-WindowsFeature Web-Server & `
     powershell -Command Add-WindowsFeature Web-Asp-Net & `
     %windir%\System32\inetsrv\appcmd set apppool /apppool.name:DefaultAppPool /managedRuntimeVersion:v2.0 & `
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\* & `
+    powershell -Command Invoke-WebRequest https://dotnetbinaries.blob.core.windows.net/servicemonitor/1342334/ServiceMonitor.exe -OutFile C:\ServiceMonitor.exe ;
```

Comment:
What does `1342334` in the download URL correspond to?  It would be nice if this were something logical such as the version number of ServiceMonitor.  This would help make the Dockerfiles more readable so the user can see which version is included.

Reply:
The 1342334 is the monotonically increasing build number. Other identifiers I could look into using instead are release number or maybe git SHA

Reply:
I would like to see the release number.  This is going to be the most meaningful to the Docker users who want to know precisely what is included in the image.

Reply:
Is this acceptable `https://dotnetbinaries.blob.core.windows.net/servicemonitor/Release-3/ServiceMonitor.exe`?

Reply:
Why not align with the Product Version - 0.3.0.1?

Reply:
The way ServiceMonitor is being built today, Product Version has no semantic meaning. It's never been revved.
https://github.com/Microsoft/IIS.ServiceMonitor/blob/9ffa751e183989ff94e033b6aa6a09e9ca2293d1/src/ServiceMonitor/ServiceMonitor.rc#L79

Reply:
Just spoke to @mcy94w offline. We're going rebuild ServiceMonitor and [bump both the file version and product version to 2.0.0.0](https://github.com/Microsoft/IIS.ServiceMonitor/issues/21)
Once I have a new build, I'll publish it to `https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.0.0/ServiceMonitor.exe` and then update this PR